src/Logging/PreformattedEchoHandler.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace SilverStripe\Logging;
+
+use Monolog\Handler\AbstractProcessingHandler;
+
+/**
+ * Echo the output as preformatted HTML, emulating console output in a browser.
+ * Tiding us over until we can properly decoupled web from CLI output.
+ * Do not use this API outside of core modules,
+ * it'll likely be removed as part of a larger refactor.
+ *
+ * See https://github.com/silverstripe/silverstripe-framework/issues/5542
+ *
+ * @experimental
```

Comment:
```suggestion
 * @internal 
```

`@internal` is part of PHPdoc but I couldn't find any detail about `@experimental` (and my IDE didn't treat it in any special way.